### DISCUSSION TITLE
Why object inside a rigid bodies (react-three-rapier) shaking when applying linear velocity?

### DISCUSSION THREAD

**Discussion by AmirHo3einJS** [pmndrs/react-three-rapier#195] [Q&A]
![203720342-13180a5b-dc2a-45b6-92fe-793a91d19005](https://user-images.githubusercontent.com/101206088/206831608-c6f5bd14-88ea-4a03-b71d-2844cd932d36.gif)

So here is an example, can someone explain to me why this is happening and how can I prevent this?
This is a next js version 13 project and I'm using react three fiber version ^8.9.1 and react three rapier ^0.10.0.

here is the code where I move my rigid body:

![203725607-017ed274-485a-41b8-9d05-cca5ac40bb61](https://user-images.githubusercontent.com/101206088/206831614-e10d08d7-01d8-449b-a56c-fd2532b16206.png)

and here is my scene:

![203726054-092a8165-c8c6-4799-8244-24e5295ba66e](https://user-images.githubusercontent.com/101206088/206831626-871593fc-cea4-4141-8fd1-b8e7243871ae.png)

**Comment by rockbrok** [comment]
Did you find an answer?
